docs/src/main/asciidoc/spring-security.adoc

```diff
@@ -0,0 +1,252 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Quarkus Extension for Spring Security API
+
+include::./attributes.adoc[]
+
+While users are encouraged to use Java standard annotations for security authorizations, Quarkus provides a compatibility layer for Spring Security in the form of the `spring-security` extension.
+
+This guide explains how a Quarkus application can leverage the well known Spring Security annotations to define authorizations on RESTful services using roles.
+
+== Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+
+== Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step.
+However, you can go right to the completed example.
+
+Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+
+The solution is located in the `spring-security-quickstart` {quickstarts-tree-url}/spring-security-quickstart[directory].
+
+== Creating the Maven project
+
+First, we need a new project. Create a new project with the following command:
+
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=spring-security-quickstart \
+    -DclassName="org.acme.spring.web.GreetingController" \
+    -Dpath="/greeting" \
+    -Dextensions="spring-web, spring-security, quarkus-elytron-security-properties-file"
+cd spring-security-quickstart
+----
+
+This command generates a Maven project with a REST endpoint and imports the `spring-web`, `spring-security` and `security-properties` extensions.
+
+For more information about `security-properties` you can check the guide of link:security-properties[quarkus-elytron-security-properties-file] extension.
+
+== GreetingController
+
+The Quarkus maven plugin automatically generated a controller with the Spring Web annotations to define our REST endpoint (instead of the JAX-RS ones used by default)
```

Comment:
```suggestion
The Quarkus Maven plugin automatically generated a controller with the Spring Web annotations to define our REST endpoint (instead of the JAX-RS ones used by default).
```